src/test/java/com/easypost/beta/ReferralTest.java

```diff
@@ -95,6 +96,7 @@ public void testAll() throws EasyPostException {
      * @throws EasyPostException when the request fails.
      */
     @Test
+    @Disabled // failing on replay
```

Comment:
Add the why, which in this case is most likely due to the `url-form encoding` crap that Stripe uses (this is the case with all other VCRs).

If that's true, we should remove the associated cassette instead of re-recording it.

Reply:
I guess we'll see if/when it fails in the C# library when we get around to adding Referral to that as well.